app/views/devise/passwords/new.html.haml

```diff
@@ -18,6 +18,6 @@
     = f.input_field :email, autofocus: true, autocomplete: 'off', class: 'form-control'
 
   .form-group
-    = f.button :submit, t('.send_button'), class: 'button'
+    = govuk_button() { t('.send_button') }
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3605/files#r526000031)